logic/build.gradle.kts

```diff
@@ -55,53 +56,53 @@ kotlin {
                 implementation(project(":util"))
 
                 // coroutines
-                implementation(Dependencies.Coroutines.core) {
-                    version {
-                        strictly(Versions.coroutines)
-                    }
-                }
-                implementation(Dependencies.Coroutines.core)
-                implementation(Dependencies.Kotlinx.serialization)
-                implementation(Dependencies.Kotlinx.dateTime)
-                implementation(Dependencies.UUID.benAsherUUID)
+                implementation(libs.coroutines.core.map {
+                    project.dependencies.create(it, closureOf<ExternalModuleDependency> {
+                        version { strictly(libs.versions.coroutines.get()) }
+                    })
+                })
+                implementation(libs.coroutines.core)
```

Comment:
I noticed recently that we have this duplication of `coroutines-core`.

No need to fix it now, I can remove it when upgrading to Kotlin 1.7.

Reply:
Great catch 🦅